File: Assets/Scripts/Killers.js
#pragma strict

function Start () {
	Time.timeScale = PlayerPrefs.GetInt("paused");
}

function OnCollisionStay(col : Collision) {
	
	if (col.collider.name == "Sphere" || col.collider.name == "Sphere #2") {
		Application.LoadLevel(Application.loadedLevel);
	}
}

File: Assets/Scripts/Killers.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class Killers : MonoBehaviour
{
    public virtual void Start()
    {
        Time.timeScale = PlayerPrefs.GetInt("paused");
    }

    public virtual void OnCollisionStay(Collision col)
    {
        if ((col.collider.name == "Sphere") || (col.collider.name == "Sphere #2"))
        {
            Application.LoadLevel(Application.loadedLevel);
        }
    }

}